### ISSUE TITLE
Make AKS-CI green

### ISSUE THREAD

**Issue by svollath** [epinio/epinio#1777]

**Comment by svollath** [comment]
When moving to a new AWS account, AKS-CI-* tests started to fail. AKS-CI is using AWS route53 for DNS.

Error:
 `An error occurred (SignatureDoesNotMatch) when calling the ChangeResourceRecordSets operation: The request signature we calculated does not match the signature you provided. Check your AWS Secret Access Key and signing method. Consult the service documentation for details.`

There's no obvious reason for route53 commands failing, as EKS and GKE tests are using the same code and secrets to make DNS records.

The error message above is from AWS, and appears for a wide range of possible errors/causes - e.g. from wrong credentials, over malformed commands to missing files, aso.

**Comment by svollath** [comment]
Yesterday I kept an azure cluster to check the setup for Epinio, and did some general comparison:

- traefik still gets an IP (not hostname) as expected for AKS, so OK
- I skipped DNS record creation from the workflow, and created those manually in route53, which made the AKS-CI continue, and successfully run acceptance tests (up to cleaning DNS records, where it then failed)
- the first AKS-CI (#562) test failing after switching the AWS creds, was running on the same Windows version (windows-2022, Version: 20220925.1) as the last AKS-CI (#561) test that passed - so this shouldn't be Windows image/version related.

**Comment by svollath** [comment]
Today was about changing various conditions for the workflow to check:

- switched to windows-2019 runner image - which also failed, so switched back to windows-latest
- moved AWS_REGION var from workflow env to "Configure AWS credentials for Route53" like with gke.yml - still fails
- Introduced "Testing AWS access" step (`aws sts get-caller-identity`), which succeded

Will continue with a function that will create a hardcoded DNS A record ...

**Comment by svollath** [comment]
I was also testing with a local aws-cli on Windows10, which was successful with the current secrets.
- I ran `aws sts get-caller-identity`, when the cli was configured with `aws configure` -> successful
- I deconfigured aws-cli, and used `setx` to hold secrets, and ran `aws sts get-caller-identity` -> successful
- Then, I was just removing the first character of the secret-key, and was able to reproduce `SignatureDoesNotMatch`
- I used setx to export the secrtes with quotes, and ran `aws sts get-caller-identity` -> successful

Then I concluded that our secret-access-key starting with a "/", ends up malformed, but just in the "Windows case".
- I introduced test secrets with the same keys, but quoted in github action secrets, but this fails with
`Run aws-actions/configure-aws-credentials@v1
Error: The security token included in the request is invalid.`

aws-actions/configure-aws-credentials@v1 fails for a leading "/" in the secret-access-key, but only in the Windows case - while Windows itself could deal with the same secret-access-key in env.

So the best way to fix this is to create a new secret-access key for epinio-ci AWS user, that doesn't start with "/", or doesn't contain any special characters.
